ra2ce/runners/direct_analysis_runner.py

```diff
@@ -37,8 +37,8 @@ def __str__(self) -> str:
     @staticmethod
     def can_run(ra2ce_input: ConfigWrapper) -> bool:
         if (
-            not ra2ce_input.analysis_config
-            or "direct" not in ra2ce_input.analysis_config.config_data
+            not bool(ra2ce_input.analysis_config)
```

Comment:
you don't really require this in python, when something is `None` it checks directly to `False`, if there's content to `True`, pretty much as with "any" collection (try doing `if []` or `if ["asdf"]` and see the differences.

Reply:
It seemed to fail without bool, but not sure why. Now it works.